.travis.yml

```diff
@@ -1,11 +1,22 @@
+---
 dist: trusty
 addons:
-    firefox: "49.0"
```

Comment:
Are the firefox test failing?

Reply:
@jordanaustin Yeah. A bunch of 'em _and_ failing upstream 😞 . Unfortunately, I didn't have the bandwidth to resolve. Since we are only targeting Chrome at the moment anyway ([called it out at the top of the README.md](https://github.com/VernierSoftwareTechnology/dom-to-image/blob/192222387defa0f09902600376822d0ca8ef8eb5/README.md)), I figured we could hold off and pull in any future fixes from upstream.